docs/quickstart/google-cloud-platform.md

```diff
@@ -0,0 +1,116 @@
+# Google Cloud Platform
+
+In this quickstart, you will learn how to deploy the Coder control plane instance, and deploy your first template. 
```

Comment:
```suggestion
In this quickstart, you will learn how to deploy the Coder control plane instance and your first template. 
```

The `,` was incorrect since the latter clause is dependent. And, we can omit `deploy` for brevity.